indico/modules/attachments/templates/attachments.html

```diff
@@ -9,6 +9,23 @@
     <div class="attachment-editor">
         <div class="action-box fixed-width">
             <div class="section">
+                <span class="icon icon-package-download"></span>
+                <div class="text">
+                    <div class="label">
+                        {% trans %}Generate a material package{% endtrans %}
+                    </div>
+                    <div>{% trans %}You can generate a file consisting of attachments in the event{% endtrans %}</div>
```

Comment:
i think "containing" fits better than "consisting of" in this case. also, missing `.` at the end of the sentence?